src/components/Experimental.astro

```diff
@@ -0,0 +1,3 @@
+<span class="bg-surface-accent-light text-content-accent text-xs px-2 py-0.5 rounded-full">
```

Comment:
@alexBinns Suggestion: Create a generic badge component with a prop called "variant" rather than a new component for each variant of badge.

The variant prop can accept semantic badge names (experimental, required, deprecated etc) and encapsulate all styling required for each variant 👍

Reply:
I have done that now.